### PR TITLE
feat(amqplib.js): add support fot amqplib

### PR DESCRIPTION
**Pull request by ranrib**

### CODE REVIEW

File: src/wrappers/amqplib.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        });
+
+        const { label, setError } = tracer;
+        // eslint-disable-next-line no-param-reassign
+        message.epsagon = {
+            label,
+            setError,
+        };
+        const { slsEvent: nodeEvent, startTime: nodeStartTime } = eventInterface.initializeEvent(
+            'node_function', 'message_handler', 'execute', 'runner'
```
Comment (galbash) [inline src/wrappers/amqplib.js:57]: try extracting the name from the `callback` before using `message_handler`. also there better be a second level which is `<queue name>-consumer` just so if there are several different consumers it's easier to distinguish between them

File: src/wrappers/amqplib.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        );
+        let runnerResult;
+        try {
+            runnerResult = callback(message);
+        } catch (err) {
+            originalHandlerSyncErr = err;
+        }
+
+        if (callback.name) {
```
Comment (galbash) [inline src/wrappers/amqplib.js:66]: ahh i see you do it here, anyway add the `<queue name>-consumer` (or something similar) option. Maybe also consult with @ronyis

File: src/wrappers/amqplib.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+            tracer.sendTrace(() => {});
+        }
+        tracer.addRunner(nodeEvent, runnerResult);
+    } catch (err) {
+        tracer.addException(err);
+    }
+    // Throwing error in case of sync user function.
+    if (originalHandlerSyncErr) {
+        throw originalHandlerSyncErr;
+    }
```
Comment (galbash) [inline src/wrappers/amqplib.js:92]: you forgot to return `runnerResult` here

File: src/wrappers/amqplib.js (hunk 1 above)
Comment (ranrib) [reply src/wrappers/amqplib.js:57]: 👍

File: src/wrappers/amqplib.js (hunk 2 above)
Comment (ranrib) [reply src/wrappers/amqplib.js:66]: 👍

File: src/wrappers/amqplib.js (hunk 3 above)
Comment (ranrib) [reply src/wrappers/amqplib.js:92]: 👍

**Pull request merged**